keps/sig-api-machinery/1281-network-proxy/README.md

```diff
@@ -403,6 +403,17 @@ Beta:
   are spending dialing out.
 - Ensure we have metrics on each existing egress use case.
 
+GA:
+
+- TBD
```

Comment:
IIUC, currently if we want to add a new kube-apiserver we need to do a rolling upgrade of all existing ones, as they need to know identity of all kube-apiservers in the fleet. Is that correct?

Can we add fixing that to GA criteria too (I believe this should be solvable with apiserver-identity: https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1965-kube-apiserver-identity which should hopefully be getting to GA relatively soon too).

Reply:
> IIUC, currently if we want to add a new kube-apiserver we need to do a rolling upgrade of all existing ones, as they need to know identity of all kube-apiservers in the fleet. Is that correct?

Yes - I suspect you're thinking of the static --server-count flag in konnectivity-server binary.

> 
> Can we add fixing that to GA criteria too (I believe this should be solvable with apiserver-identity: https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1965-kube-apiserver-identity which should hopefully be getting to GA relatively soon too).

I added https://github.com/kubernetes-sigs/apiserver-network-proxy/issues/358 to this draft GA requirements. In a recent discussion of approaches, we decided KAS identity is not the correct approach for konnectivity-server because it should not be assumed 1:1 with KAS. (But a similar approach could be reasonable, like konnectivity-server specific identity / state).

Reply:
> Yes - I suspect you're thinking of the static --server-count flag in konnectivity-server binary.

yes - thanks!

> we decided KAS identity is not the correct approach for konnectivity-server because it should not be assumed 1:1 with KAS. (But a similar approach could be reasonable, like konnectivity-server specific identity / state).

Fair - I would primarily the problem to be addressed, I'm definitely open for other options.

Thanks for adding!